.gitmodules

```diff
@@ -18,4 +18,4 @@
 	url = https://github.com/msgpack/msgpack-c.git
 [submodule "vendor/backward-cpp"]
 	path = vendor/backward-cpp
-	url = https://github.com/bombela/backward-cpp.git
+	url = https://github.com/n-hutton/backward-cpp.git
```

Comment:
If we're forking a 3rd party lib, then it should probably live under the fetchai org.

Reply:
This is fine for the moment